pkg/aggregate/functions.go

```diff
@@ -75,14 +75,15 @@ func (a *FloatAggregator) UpdateExpr(col string, bucket int) string {
 }
 
 func (a *FloatAggregator) InitExpr(col string, buckets int) string {
-	return fmt.Sprintf("_%s_%s=init_array(%d,'double');", col, a.attr, buckets)
+	// TODO: consider adding default init value per aggregator
+	return fmt.Sprintf("_%s_%s=init_array(%d,'double', %f);", col, a.attr, buckets, a.val)
```

Comment:
Let's be consistent on spacing.

Reply:
it's `go fmt ../../...` I have nothing to do with that

Reply:
I mean spacing within the update expression - that's not `go fmt`'s fault :)

Reply:
it so negligible ...